### PR TITLE
Fix Compiling on Linux

### PR DESCRIPTION
**Pull request by Hibyehello**

### CODE REVIEW

**Comment by malleoz** [comment]
LGTM

**Review by malleoz — APPROVED** [review]

File: Source/Core/DolphinTool/ToolHeadlessPlatform.cpp [hunk 1]
```diff
@@ -84,6 +84,11 @@ bool Host_RendererIsFullscreen()
   return false;
 }
 
+bool Host_TASInputHasFullFocus()
+{
+  return false;
+}
+
```
Comment (vabold) [inline Source/Core/DolphinTool/ToolHeadlessPlatform.cpp:91]: What does this fix?

File: Source/Core/Core/Config/MainSettings.h [hunk 2]
```diff
@@ -232,12 +232,12 @@ extern const Info<std::string> MAIN_IMGUI_FONT;
...
 enum class OutlineRes
 {
-  None,
+  Off,
   Thin,
   Thick,
   Full
 };
-extern const Info<OutlineRes> MAIN_IMGUI_OUTLINE_RES;
+extern const Info<Config::OutlineRes> MAIN_IMGUI_OUTLINE_RES;
```
Comment (vabold) [inline Source/Core/Core/Config/MainSettings.h:240]: Why the namespace change? Is it consistent with the rest of the file?

**Review by vabold — DISMISSED** [review]

**Comment by vabold** [comment]
(These were meant to be comments, not change requests.)

File: Source/Core/Core/Config/MainSettings.h (hunk 2 above)
Comment (Hibyehello) [reply Source/Core/Core/Config/MainSettings.h:240]: Linux was complaining about it, but I still don't know why

Comment (Hibyehello) [reply Source/Core/Core/Config/MainSettings.h:240]: Issue #58 has the error that occurs without it

Comment (Hibyehello) [reply Source/Core/Core/Config/MainSettings.h:240]: Also it's not consistent with the rest of the file which is why I'm confused about the error

**Pull request merged**